src/views/Scenario/ScenarioHook.js

```diff
@@ -17,7 +19,10 @@ export const useScenario = () => {
   const currentScenarioData = useCurrentScenarioData();
   const organizationId = useOrganizationId();
   const workspaceId = useWorkspaceId();
-
+  const datasets = useWorkspaceDatasets();
+  const currentScenarioDatasetName = useMemo(() => {
+    return datasets?.find((dataset) => dataset.id === currentScenarioData?.datasetList[0])?.name;
```

Comment:
Not sure if we want to hide it when we don't find the dataset id in the list, but we could add a fallback value here (e.g. `N/A`, `Not found`, ...) to explain that we didn't find the dataset id? (e.g. if the dataset has been deleted).

Also, the part `datasetList[0]` may crash the webapp in the specific case of scenarios that have been created manually without any datasets. Maybe we should display a different message in this case (e.g. `No dataset`) or hide the dataset name label in this case?